ebusd-2.1.x/en/vaillant/15.ctlv2.csv

```diff
@@ -0,0 +1,387 @@
+w,,z2Timer.Monday,timer heating monday,,,,00,,,wTimeSlotWithTemp,,,
+w,,z2Timer.Tuesday,timer heating tuesday,,,,01,,,wTimeSlotWithTemp,,,
+w,,z2Timer.Wednesday,timer heating wednesday,,,,02,,,wTimeSlotWithTemp,,,
+w,,z2Timer.Thursday,timer heating thursday,,,,03,,,wTimeSlotWithTemp,,,
+w,,z2Timer.Friday,timer heating friday,,,,04,,,wTimeSlotWithTemp,,,
+w,,z2Timer.Saturday,timer heating saturday,,,,05,,,wTimeSlotWithTemp,,,
+w,,z2Timer.Sunday,timer heating sunday,,,,06,,,wTimeSlotWithTemp,,,
+# timers zone 3,,,,,,,,,,,,,
+*r,,,,,,B555,a50200,,,,,,
+*w,,,,,,B555,a60200,,,,,,
+r,,z3Timer.Config,timer Configuration(static atm? describes featue set?),,,B555,a30200,,,HEX:9,,,Configuration
+r,,z3Timer.Timeframes,timer Anzahl,,,B555,a40200,,,slotCountWeek,,,Amount of configured slots per weekday
+r,,z3Timer.Monday0,timer heating monday 1,,,,0000,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Monday1,timer heating monday 2,,,,0001,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Monday2,timer heating monday 3,,,,0002,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Tuesday0,timer heating tuesday 1,,,,0100,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Tuesday1,timer heating tuesday 2,,,,0101,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Tuesday2,timer heating tuesday 3,,,,0102,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Wednesday0,timer heating wednesday 1,,,,0200,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Wednesday1,timer heating wednesday 2,,,,0201,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Wednesday2,timer heating wednesday 3,,,,0202,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Thursday0,timer heating thursday 1,,,,0300,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Thursday1,timer heating thursday 2,,,,0301,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Thursday2,timer heating thursday 3,,,,0302,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Friday0,timer heating friday 1,,,,0400,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Friday1,timer heating friday 2,,,,0401,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Friday2,timer heating friday 3,,,,0402,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Saturday0,timer heating saturday 1,,,,0500,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Saturday1,timer heating saturday 2,,,,0501,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Saturday2,timer heating saturday 3,,,,0502,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Sunday0,timer heating sunday 1,,,,0600,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Sunday1,timer heating sunday 2,,,,0601,,,rTimeSlotWithTemp,,,
+r,,z3Timer.Sunday2,timer heating sunday 3,,,,0602,,,rTimeSlotWithTemp,,,
+w,,z3Timer.Monday,timer heating monday,,,,00,,,wTimeSlotWithTemp,,,
+w,,z3Timer.Tuesday,timer heating tuesday,,,,01,,,wTimeSlotWithTemp,,,
+w,,z3Timer.Wednesday,timer heating wednesday,,,,02,,,wTimeSlotWithTemp,,,
+w,,z3Timer.Thursday,timer heating thursday,,,,03,,,wTimeSlotWithTemp,,,
+w,,z3Timer.Friday,timer heating friday,,,,04,,,wTimeSlotWithTemp,,,
+w,,z3Timer.Saturday,timer heating saturday,,,,05,,,wTimeSlotWithTemp,,,
+w,,z3Timer.Sunday,timer heating sunday,,,,06,,,wTimeSlotWithTemp,,,
+# includes,,,,,,,,,,,,,
+!include,errors.inc,,,,,,,,,,,,
```

Comment:
number of lines between en and de is different but should be identical

Reply:
fixed now, merged PR from @mklooss